tests/sites/test_primary_sidebar_toc/sec/index.rst

```diff
@@ -0,0 +1,10 @@
+===============
+Section at Root
+===============
+
+Some text.
+
+.. toctree::
+   :hidden:
```

Comment:
I don't think this test case is at all dependent on the change made in this PR, but it seems like a nice test case to have in the test suite. :smile: